src/main/java/org/openstreetmap/atlas/geography/sharding/SlippyTile.java

```diff
@@ -385,30 +341,6 @@ public String toString()
     }
 
     /**
-     * Add the siblings and parent.
-     *
-     * @param candidates
-     *            The candidate tiles
-     * @param visitedTiles
-     *            The tiles already visited
-     * @param targetTile
-     *            The target tile
-     */
-    private void addNextZoomLevelTilesToCandidateList(final Queue<SlippyTile> candidates,
```

Comment:
I'd suggest keeping the old method (zoom-level irrespective) around in case it can be useful, perhaps renaming it as `getNeighborsForAllZoomLevels`.

Reply:
ok, let me do that